### ISSUE TITLE
Softmax in the evaluation stage

### ISSUE THREAD

**Issue by saeedizadi** [bodokaiser/piwise#7]
Hi, 

I've found that you are explicitly applying _log_softmax_ in your **criterion.py** file. Isn't is necessary to explicitly apply the _softmax_ function on the output of the network in the evaluation stage? that is, before getting the maximum index for each class? 

Thanks

**Comment by bodokaiser** [comment]
Hey,

yes, I think you are right. Could you test what happens if you add `F.log_softmax(labels)` [here](https://github.com/bodokaiser/piwise/blob/master/main.py#L101) and make a PR out of it?
